samples/vision_async_batch_annotate_images_beta.js

```diff
@@ -0,0 +1,69 @@
+/**
```

Comment:
Couple of nits with the name: 
- presumably this feature ain't going to be in beta forever :) Can we just drop the `_beta` from the filename?  
- we know this is a vision sample - do we need to have `vision_` in the name?

Reply:
Done.